ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml

```diff
@@ -0,0 +1,37 @@
+binary_build_commands: make commands
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+canonical_go_repository: cloud.ibm.com/cloud-provider-ibm
+promotion:
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: e2e-aws
```

Comment:
I assume all this AWS stuff is required until we have our own E2E?  If so, can we add a TODO comment to fix this?